web/app.js

```diff
@@ -396,15 +396,15 @@ var PDFViewerApplication = {
   },
 
   get pagesCount() {
-    return this.pdfDocument.numPages;
+    return this.pdfDocument ? this.pdfDocument.numPages : 0;
   },
 
   set page(val) {
-    this.pdfLinkService.page = val;
+    this.pdfViewer.currentPageNumber = val;
   },
 
   get page() { // TODO remove
-    return this.pdfLinkService.page;
+    return this.pdfViewer.currentPageNumber;
```

Comment:
Yeah, I'm not entirely sure why that comment is there in the first place, and we probably should keep the `page` getter/setter around for backwards compatibility.
I'll just remove the comment; thanks for pointing this out!